dockerfiles/piraeus-server/Dockerfile

```diff
@@ -45,7 +45,7 @@ COPY entry.sh /usr/bin/piraeus-entry.sh
 ARG ARCH
 ARG K8S_AWAIT_ELECTION_VERSION
 
-RUN wget https://github.com/LINBIT/k8s-await-election/releases/download/${K8S_AWAIT_ELECTION_VERSION}/k8s-await-election-${K8S_AWAIT_ELECTION_VERSION}-linux-${ARCH}.tar.gz -O - | tar -xvz -C /usr/bin/
+RUN wget https://github.com/LINBIT/k8s-await-election/releases/download/${K8S_AWAIT_ELECTION_VERSION}/k8s-await-election-${K8S_AWAIT_ELECTION_VERSION}-linux-$(dpkg --print-architecture).tar.gz -O - | tar -xvz -C /usr/bin/
```

Comment:
I would have NACKed this one. The Debian architecture ist the same as the GOARCH most of the time, but not always. It should not matter for amd64 and arm64, but IIRC it is GOARCH ppc64le and and Debian ppce64el. So if there is no rel reason for that, @WanzenBug please revert that back.

Reply:
I think the change was made because we use buildkit (docker buildx build), to build all architectures at once. Now we need to get the GOARCH from somewhere. You are right that this just happen to work in this case, so we should probably change it to something that always works. I will have to think about how we can achieve that.

Reply:
on it

Reply:
FYI: I think you can use `$TARGETPLATFORM`: https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images (since docker is written in go, DOCKERARCH==GOARCH). That's hopefully stable enough.

Reply:
almost all the interesting ones match (amd64, s390x, arm64). The only interesting one that does not is ppc64{el,le}. So I'm perfectly fine if we use `dpkg --print-architecture`, but pipe it throuhg some `sed`, or `case/esac` or whatever.

Reply:
> FYI: I think you can use `$TARGETPLATFORM`: https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images (since docker is written in go, DOCKERARCH==GOARCH). That's hopefully stable enough.

sounds even better than the shell method :+1:

Reply:
`TARGETARCH` is indeed working, I'm building a `ppc64` image for testing, expect a PR soon

Reply:
See #86 
Incidentally, the debian packages `linstor-controller`, `linstor-satellite`, `linstor-common` and `linstor-client` do not exist for `ppc64le`

Reply:
that is intended, and probably will stay like this. I'd guess that ppc64le, if ever, will be handled like s390x now and that it would be an arch that we only provide to our customers. I just wanted to have it as correct as possible. like if someone forks it and then provides their own ppc64le repo or whatever.